lib/client.js

```diff
@@ -182,6 +182,54 @@ Client.prototype._defineMethod = function(method, location) {
   };
 };
 
+Client.prototype._isSequenceRequired = function(methodName) {
+  var tns = this.wsdl.definitions.$targetNamespace;
+  var methodRequestName = _.result(this.wsdl.definitions, 'messages.'+methodName+'.$name');
```

Comment:
It would be nice if we had this failing the build as part of linting too (for another PR if needed).